nanodbc/nanodbc.cpp

```diff
@@ -1746,7 +1746,10 @@ class statement::statement_impl
         return cols;
     }
 
-    void reset_parameters() noexcept { NANODBC_CALL(SQLFreeStmt, stmt_, SQL_RESET_PARAMS); }
+    void reset_parameters() noexcept {
```

Comment:
```suggestion
    void reset_parameters() noexcept
    {
```
Please, stick to the existing style and formatting of the code.

Please, ensure your code conforms to our  `clang-format` specification, otherwise CI build job on Travis CI will fail, see https://travis-ci.org/nanodbc/nanodbc/jobs/600185392